languages/clojure/exercises/concept/booleans/.docs/hints.md

```diff
@@ -0,0 +1,19 @@
+## 1. Check if a fast attack can be made
```

Comment:
As per [the docs[(https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd):

> The hints should not spell out the solution, but instead point to a resource describing the solution (e.g. linking to documentation for the function to use).

Applying that to this hints.md file, maybe you could come with a way to not list the actual operator that students should be using? That said, in this particular case I don't really mind it that much to be honest.